brave/brave-core
# [Brave News]: FeedV2 should provide notifications when new data is available

Resolves https://github.com/brave/brave-browser/issues/34819

[Screencast from 2023-12-12 15-11-47.webm](https://github.com/brave/brave-core/assets/7678024/443df7e6-b5fb-4336-b175-17f43d645ee4)

Note: This button will only show up if
1. Feeds have changed
2. Data on the server has changed



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Scroll down to the Brave News feed
2. Click the Customize button
3. Subscribe to a feed - after a short delay the `New content available` button should show up
4. Unsubscribe from the feed - the button should disappear again
5. Subscribe to the feed again
6. Close the customize dialog
7. Click the button - the feed should reload and the button should disappear.
